test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -298,21 +297,21 @@ def test_execjob_end_reject_request(self):
         self.server.expect(JOB, {'job_state': "R"}, id=jid)
 
         # Wait for job to be in E state
-        time.sleep(30)
-        self.server.expect(JOB, {'job_state': "E"}, id=jid, offset=20)
+        time.sleep(15)
+        hook_execution_time = int(time.time())
+        self.server.expect(JOB, {'job_state': "E"}, id=jid, offset=10)
```

Comment:
currently you are waiting total of 35s however the job is only 30s. any reason why?
Also a recommendation, you can either increase the sleep to 20s or update the offset to be 20s.

Reply:
Sorry @Klovely I think I miscalculated. I do see you are only sleeping for 30s. I am fine with the changes.